frontend/.opencode/skills/vue-best-practices/rules/path-alias-vue-sfc.md

```
@@ -0,0 +1,91 @@
+
+**Step 1: Use path.resolve for aliases**
+```typescript
+// vite.config.ts
+import { defineConfig } from 'vite'
+import vue from '@vitejs/plugin-vue'
+import path from 'path'
+
+export default defineConfig({
+  plugins: [vue()],
+  resolve: {
+    alias: {
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)